ui/imports/utils/Constants.qml

```diff
@@ -149,6 +149,41 @@ QtObject {
         ]
     }
 
+    readonly property QtObject settingsSection: QtObject {
+        readonly property int itemSpacing: 10
+        readonly property int mainHeaderFontSize: 28
+
+//        readonly property int radius: 8
```

Comment:
why is this commented? can it be removed, or still needed?

Reply:
@alaibe thanks for this, it's leftover, removed now.